packages/core/stories/overlay/overlay.stories.tsx

```diff
@@ -110,12 +110,10 @@ export const LongContent = () => {
       <OverlayPanel
         style={{
           width: 300,
-          height: 200,
```

Comment:
This feels a worth callout?

Reply:
I think this would still work?

Reply:
i don't know, didn't try. Just thinking if that's the code change needed to be done, then we need to mention it

Reply:
it is not changing the behaviour, but if the user sets height here then the paddings are not respected, so i just moved it to the content so the overflow is in the content, not in the panel and improve the example... but this would still work as before